src/homekit.cpp

```diff
@@ -23,13 +23,17 @@ void target_door_state_set(const homekit_value_t new_value);
 homekit_value_t obstruction_detected_get();
 homekit_value_t active_state_get();
 
+// Make device_name available
+extern "C" char device_name[16];
+
 /********************************** MAIN LOOP CODE *****************************************/
 
 void homekit_loop() {
     arduino_homekit_loop();
 }
 
 void setup_homekit() {
+    sprintf(device_name, "ratgdo %X\0", ESP.getChipId());
```

Comment:
Since the physical thing being controlled is a garage door, let's keep the name `Garage Door XXXXXXXX`.

Also, consider fully-specifying the `%X` so leading zeroes don't get eaten and the size is bounded.

Reply:
Changed to "Garage Door XXXXXX" and limited to 6 hex ints. Initially I thought `.getChipID()` would return the entire MAC address, but turns out [it only returns the last 3 bytes](https://github.com/esp8266/Arduino/issues/921).